### PR TITLE
LPS-173785 [Raylife D2C/AP] Create Raylife Data Objects  Sales Goals Inserting Data

### PR DESCRIPTION
**Pull request by EugenioAntunes**
[LPS-162771](https://issues.liferay.com/browse/LPS-162771)

### CODE REVIEW

**Comment by EugenioAntunes** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/object-entries/6_sales-goal.json [hunk 1]
```diff
@@ -419,6 +419,3114 @@
 			"initialReferenceDate": "2022-12-01T00:00:00",
 			"productExternalReferenceCode": "RAYAP-001",
 			"r_userToSalesGoals_userId": "20125"
+		},
+		{
+			"finalReferenceDate": "2022-12-31T00:00:00Z",
+			"goalValue": "20000",
+			"initialReferenceDate": "2022-12-01T00:00:00",
+			"productExternalReferenceCode": "RAYA001",
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/object-entries/6_sales-goal.json:427]: This ERC does not exist. Please change it to "RAY001".

File: modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/object-entries/6_sales-goal.json [hunk 2]
```diff
@@ -419,6 +419,3114 @@
...
+			"goalValue": "15000",
+			"initialReferenceDate": "2021-11-01T00:00:00",
+			"productExternalReferenceCode": "RAYA003",
+			"r_userToSalesGoals_userId": "20125"
+		},
+		{
+			"finalReferenceDate": "2021-10-31T00:00:00Z",
+			"goalValue": "10000",
+			"initialReferenceDate": "2021-10-01T00:00:00",
+			"productExternalReferenceCode": "RAYA003",
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/object-entries/6_sales-goal.json:861]: This ERC does not exist. Please change it to "RAY003".

File: modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/object-entries/6_sales-goal.json [hunk 3]
```diff
@@ -419,6 +419,3114 @@
...
+			"goalValue": "11000",
+			"initialReferenceDate": "2021-02-01T00:00:00",
+			"productExternalReferenceCode": "RAYA002",
+			"r_userToSalesGoals_userId": "20125"
+		},
+		{
+			"finalReferenceDate": "2021-01-31T00:00:00Z",
+			"goalValue": "19000",
+			"initialReferenceDate": "2021-01-01T00:00:00",
+			"productExternalReferenceCode": "RAYA002",
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/object-entries/6_sales-goal.json:756]: This ERC does not exist. Please change it to "RAY002".

File: modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/object-entries/6_sales-goal.json [hunk 4]
```diff
@@ -419,6 +419,3114 @@
...
+			"goalValue": "15000",
+			"initialReferenceDate": "2022-11-01T00:00:00",
+			"productExternalReferenceCode": "RAYA004",
+			"r_userToSalesGoals_userId": "20125"
+		},
+		{
+			"finalReferenceDate": "2022-10-31T00:00:00Z",
+			"goalValue": "10000",
+			"initialReferenceDate": "2022-10-01T00:00:00",
+			"productExternalReferenceCode": "RAYA004",
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/object-entries/6_sales-goal.json:945]: This ERC does not exist. Please change it to "RAY004".

File: modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/object-entries/6_sales-goal.json [hunk 5]
```diff
@@ -419,6 +419,3114 @@
...
+		{
+			"finalReferenceDate": "2022-12-31T00:00:00Z",
+			"goalValue": "20000",
+			"initialReferenceDate": "2022-12-01T00:00:00",
+			"productExternalReferenceCode": "RAYA001",
+			"r_userToSalesGoals_userId": "20125"
+		},
+		{
+			"finalReferenceDate": "2022-11-31T00:00:00Z",
+			"goalValue": "15000",
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/object-entries/6_sales-goal.json:432]: This value should be a number. Please change it.

File: modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/object-entries/6_sales-goal.json [hunk 6]
```diff
@@ -419,6 +419,3114 @@
...
+			"initialReferenceDate": "2022-06-01T00:00:00",
+			"productExternalReferenceCode": "RAYA001",
+			"r_userToSalesGoals_userId": "20125"
+		},
+		{
+			"finalReferenceDate": "2022-05-31T00:00:00Z",
+			"goalValue": "6000",
+			"initialReferenceDate": "2022-05-01T00:00:00",
+			"productExternalReferenceCode": "RAYA001",
+			"r_userToSalesGoals_userId": "20125"
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/object-entries/6_sales-goal.json:477]: Maybe you should use a number here.

**Review by joanagrego — CHANGES_REQUESTED** [review]
Good job! I added some suggestions, please pay more attention to the ERCs.

File: modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/object-entries/6_sales-goal.json (hunk 1 above)
Comment (EugenioAntunes) [reply modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/object-entries/6_sales-goal.json:427]: Done

File: modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/object-entries/6_sales-goal.json (hunk 2 above)
Comment (EugenioAntunes) [reply modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/object-entries/6_sales-goal.json:861]: Done

File: modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/object-entries/6_sales-goal.json (hunk 1 above)
Comment (EugenioAntunes) [reply modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/object-entries/6_sales-goal.json:427]: Done

File: modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/object-entries/6_sales-goal.json (hunk 5 above)
Comment (EugenioAntunes) [reply modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/object-entries/6_sales-goal.json:432]: Done

File: modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/object-entries/6_sales-goal.json (hunk 4 above)
Comment (EugenioAntunes) [reply modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/object-entries/6_sales-goal.json:945]: Done

File: modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/object-entries/6_sales-goal.json (hunk 3 above)
Comment (EugenioAntunes) [reply modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/object-entries/6_sales-goal.json:756]: Done

**Review by claricebrasil — APPROVED** [review]

**Comment by EugenioAntunes** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/object-entries/6_sales-goal.json [hunk 7]
```diff
@@ -2,423 +2,3531 @@
...
+			"goalValue": 20500,
+			"initialReferenceDate": "2022-12-01T00:00:00",
+			"productExternalReferenceCode": "RAYA001",
+			"r_userToSalesGoals_userId": 20125
+		},
+		{
+			"finalReferenceDate": "2022-11-31T00:00:00Z",
+			"goalValue": 15500,
+			"initialReferenceDate": "2022-11-01T00:00:00",
+			"productExternalReferenceCode": "RAYA001",
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/object-entries/6_sales-goal.json:2030]: Hey,  you forgot to fix it. Please change it to "RAY001". Same for the others.

**Review by joanagrego — CHANGES_REQUESTED** [review]
Hey, you forgot to the adjustments in the ERCs. Please take a look at it.

File: modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/object-entries/6_sales-goal.json (hunk 7 above)
Comment (EugenioAntunes) [reply modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/object-entries/6_sales-goal.json]: Done

**Review by joanagrego — APPROVED** [review]
Approved! Thank you @EugenioAntunes :white_check_mark:

**Comment by joanagrego** [comment]
ci:test:bundle

**Comment by joanagrego** [comment]
ci:forward

**Comment by EugenioAntunes** [comment]
ci:test:sf

**Comment by EugenioAntunes** [comment]
ci:forward

**Pull request closed without merging**